benchmark/mprof.sh

```diff
@@ -0,0 +1,11 @@
+export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
```

Comment:
What about adding "set -vex" (or a subset of it) to make sure it runs correctly?